azure-pipelines.yml

```
@@ -0,0 +1,166 @@
+variables:
+  CurrentSemanticVersionBase: '0.0.1'
+  PreviewNumber: $[counter(variables['CurrentSemanticVersionBase'], 1001)]
+  CurrentSemanticVersion: '$(CurrentSemanticVersionBase)-preview$(PreviewNumber)'
+  NugetPackageVersion: '$(CurrentSemanticVersion)'
+  #MONO_VERSION: 6_4_0
+  #XCODE_VERSION: 11.4
+  NET_VERSION: '6.0.x'
+  RunPoliCheck: 'false'
+  PathToSolution: 'src/CommunityToolkit.Maui.sln'
+  PathToCommunityToolkitCsproj: 'src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj'
+  PathToCommunityToolkitUnitTestCsproj: 'src/CommunityToolkit.Maui.UnitTests/CommunityToolkit.Maui.UnitTests.csproj'
+  provisionator.vs2019: '$(System.DefaultWorkingDirectory)/eng/provisioning/vs2019.csx'
+
+resources:
+  repositories:
+    - repository: internal-templates
+      type: github
+      name: xamarin/yaml-templates
+      endpoint: xamarin
+      ref: refs/heads/main
+
+trigger:
+  branches:
+    include:
+    - main
+    - develop
+  tags:
+    include:
+    - '*'
+  paths:
+    exclude:
+    - README.md
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - main
+    - develop
+  paths:
+    exclude:
+    - README.md
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - develop
+
+jobs:      
+  - job: build_windows
+    displayName: Build Windows Library
+    pool:
+      vmImage: windows-latest
+    steps:
+      - task: UseDotNet@2
+        displayName: 'Install .NET SDK'
+        inputs:
+          version: '$(NET_VERSION)'
+          includePreviewVersions: true
+      - task: xamops.azdevex.provisionator-task.provisionator@1
+        displayName: 'Provision VSWindows'
+        inputs:
+          provisioning_script: $(provisionator.vs2019)
+
+      - task: CmdLine@2
+        displayName: 'Clear Local NuGet Cache' #https://github.com/actions/virtual-environments/issues/1090#issuecomment-748452120
+        inputs:
+          script: 'nuget locals all -clear'
+      - task: CmdLine@2
+        displayName: 'Clean Solution'
+        inputs:
+          script: 'dotnet clean $(PathToSolution)'
+      # if this is a tagged build, then update the version number
+      - powershell: |
+          $buildSourceBranch = "$(Build.SourceBranch)"
+          $tagVersion = $buildSourceBranch.Substring($buildSourceBranch.LastIndexOf("/") + 1)
+          Write-Host("Branch = $buildSourceBranch, Version = $tagVersion");
+          Write-Host ("##vso[task.setvariable variable=NugetPackageVersion;]$tagVersion")
+        displayName: Set NuGet Version to Tag Number
+        condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
+      # restore, build and pack the packages
+      - task: CmdLine@2
+        displayName: Build CommunityToolkit.Maui.csproj
```

```suggestion
        displayName: 'Build Community Toolkit'
```
Making consistent with the lower job, also just needed to trigger the CI... 😋